src/Node/Render.php

```diff
@@ -9,13 +13,31 @@
 use Twig_Node_Expression_Constant;
 use Twig_Node_Include;
 
+/**
+ * Class to create and compile a new 'Render' node Twig template.
+ *
+ * Components are rendered within their own context. The default variables are
+ * set from the component config file if not given explicitly via the tag.
+ * Variants must be referenced with a double hyphen and be configured in the main
+ * component config. – Having a separate variant file is currently not supported.
+ */
```

Comment:
In a first reading of it, my dobuts:
 * What is a context?
 * Which config file are you talking about?
 * What variant is?
Maybe we should refer here the whole `fractal variants` thing to explain what are we adding support for. Also a `@see` with it will make sense.